services/tables/src/main/java/com/linkedin/openhouse/tables/api/validator/impl/PoliciesSpecValidator.java

```diff
@@ -86,9 +87,59 @@ public boolean validate(
         return false;
       }
     }
+
+    if (policies != null
+        && policies.getReplication() != null
+        && policies.getReplication().getConfig() != null) {
+      // invalid cases for replication config
+      return policies.getReplication().getConfig().stream()
+          .allMatch(
+              replicationConfig -> {
+                if (replicationConfig.getInterval() != null) {
+                  if (!validateReplicationInterval(replicationConfig)) {
+                    failureMessage =
+                        String.format(
+                            "Replication interval provided is not a multiple of 12 or not in the range of 12h to 72h for table %s",
+                            tableUri);
+                    return false;
+                  }
+                }
+                if (replicationConfig.getDestination() != null) {
+                  if (!validateReplicationCluster(replicationConfig, tableUri)) {
+                    failureMessage =
+                        String.format(
+                            "Replication destination cluster provided cannot be equal to source cluster for table %s",
+                            tableUri);
+                    return false;
+                  }
+                }
+                return true;
+              });
+    }
     return true;
   }
 
+  /**
+   * Validate that the optional interval parameter provided by users exists as an interval of 12
+   * from 12h to 72h
+   */
+  protected boolean validateReplicationInterval(ReplicationConfig replicationConfig) {
+    int interval =
+        Integer.parseInt(
+            replicationConfig
+                .getInterval()
+                .substring(0, replicationConfig.getInterval().length() - 1));
+    return interval % 12 == 0 && interval >= 12 && interval <= 72;
+  }
+
+  /**
+   * Validate that the destination cluster provided by users is not the same as the source cluster
+   */
+  protected boolean validateReplicationCluster(
+      ReplicationConfig replicationConfig, TableUri tableUri) {
+    return !replicationConfig.getDestination().equals(tableUri.getClusterId());
```

Comment:
will probably need to configure a mapper that maps clusterId to real cluster name